ta/os_test_lib/include/os_test_lib.h

```diff
@@ -0,0 +1,31 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2018, Linaro Limited
+ */
+
+#ifndef _OS_TEST_LIB_H_
+#define _OS_TEST_LIB_H_
+
+#ifdef CFG_DYNLINK_TA
+
+int os_test_shlib_add(int a, int b);
+void os_test_shlib_panic(void);
+
+#else
+
+#include <compiler.h>
+#include <tee_internal_api.h>
+
+static int os_test_shlib_add(int a __unused, int b __unused)
+{
+	TEE_Panic(0);
```

Comment:
So if have haven't enabled `CFG_DYNLINK_TA` then this (and `os_test_shlib_panic`) test should bail out? I.e., the `ifdef` around `xtest_tee_test_1019` should ensure that this never will be called, but in case someone does so anyways from somewhere, then we just bail out? Is that the intention here?

Reply:
Exactly.

Reply:
Ok, with that you have my RB.